src/main/java/org/tron/common/runtime/Runtime.java

```diff
@@ -489,6 +489,7 @@ public void go() {
           long callValue = TransactionCapsule.getCallValue(trx.getRawData().getContract(0));
           if (callValue > 0) {
```

Comment:
![Code Smell](https://sonarsource.github.io/sonar-github/code_smell.png 'Code Smell') Code Smell: Move this left curly brace to the beginning of next line of code. ([squid:LeftCurlyBraceStartLineCheck](https://sonarcloud.io/organizations/tron-zhaohong/rules#rule_key=squid%3ALeftCurlyBraceStartLineCheck))

[See it in SonarCloud](https://sonarcloud.io/project/issues?id=java-tron&pullRequest=1633&issues=AWak_C2vdPPRVdwyhXML&open=AWak_C2vdPPRVdwyhXML)

[ISSUE_KEY]: AWak_C2vdPPRVdwyhXML